CMakeLists.txt

```diff
@@ -118,7 +118,15 @@ else()
   endif()
 endif()
 
+get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
+
+if ("${LIB64}" STREQUAL "TRUE")
+  set(LIBSUFFIX 64)
```

Comment:
A name more commonly used in different CMake packages is `LIB_SUFFIX`, and it needs to be configurable. Alternatively, CMake has a built-in `GNUInstallDirs` package that handles paths correctly for different systems and guarantees portability, unlike hardcoding paths.

Reply:
I'll take a better look at it when I'm not bogged by homework. There's a reason I haven't tagged a new release. If you wouldn't mind fixing it up and making a merge request that'd be appreciated.